### DISCUSSION TITLE
打开蓝灯后电脑右下蓝灯图标消失

### DISCUSSION THREAD

**Discussion by ToSsGirL06** [getlantern/lantern-forum-cn#407] [Q&A]
最近打开APP后，右下蓝灯标记是灰的，每次鼠标光标放到图标上图标就自动消失了，完全打不开蓝灯的网页设置。
现在电脑版的打不开APP，还请告知如何修复
如果可以请邮箱联系我方便沟通，谢谢30226237@qq.com

**Comment by MaxMKMax** [comment]
@ToSsGirL06 

您好. 请问您下载最新版本了吗? 7.8.4

https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern

**Reply by ToSsGirL06** [reply]
为此我昨天发问题前专门重新安装了一遍APP，下的最新版本。感觉出现这个问题应该就是在4月1日左右，现在除了点击APP打开后看右下角是空的啥也不能操作，因为鼠标一放到图标上图标就消失。任务管理器中打开APP时显示程序启动了，然后没几秒就消失了。
前两天我在不连VPN的情况下也不能上网了，就按照网上的操作：Internet 选项-连接-局域网设置里勾选了自动检测设置，然后能上网。跟这件事有关么？

**Reply by MaxMKMax** [reply]
@ToSsGirL06 
这事情很玄. 我们需要log来查看是什么问题. 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题.

**Comment by MaxMKMax** [comment]
@ToSsGirL06 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Comment by ToSsGirL06** [comment]
我现在的情况是打不开蓝灯网络连接的网页，因为鼠标一放到APP图标上，图标就消失了。现在我能做的仅仅是在开始里打开APP还连不上网这个情况。&nbsp;


Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年4月7日(星期天) 下午3:38
收件人: ***@***.***&gt;; 
抄送: "Jason ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)














 
@ToSsGirL06 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。
 
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
  
go to Support (转到支持)
 
select Report an Issue (选择报告问题)
 
choose issue type (选择问题类型)
 
include additional details (包括额外的详细信息)
 
hit Send Report (点击发送报告)
  
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.
 
有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
 我们将非常感谢您。

**Comment by Lion-Chan** [comment]
存在同样问题, 7.8.3和7.8.5均无法正常启动

**Reply by Lion-Chan** [reply]
[lantern.log](https://github.com/getlantern/lantern-forum-cn/files/14900081/lantern.log)

**Reply by MaxMKMax** [reply]
@Lion-Chan 
好的. 谢谢您的log.

**Comment by ToSsGirL06** [comment]
你好像没弄清楚现在的情况，我现在没法打开蓝灯，因为我一打开，当我想去右下角点图标的时候图标会消失，那么我该如何上应用程序给你提交log？我连程序都打不开怎么提交？如果说是我理解错了，还请你尽可能详细的告知如何操作，谢谢。我是付费用户，但是我快半个月没法使用你们的产品了，还请尽快帮忙协助处理。




Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月8日(星期一) 晚上11:27
***@***.***&gt;;
抄送:&nbsp;"Jason ***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)





 
@ToSsGirL06
 这事情很玄. 我们需要log来查看是什么问题. 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题.

**Reply by MaxMKMax** [reply]
@ToSsGirL06 嗯嗯好的. 那请您告诉我您用的是哪一个系统? Window xx? Mac OS? Android? IOS xx?
系统版本. 然后哪一个Lantern版本? 我也只能用这些信息帮您提交问题来修复. 

但一般只需要去按帐号(account), 然后支持 (support) 就能提交报告/LOG了. 

如果这里也闪退那就没事. 我看是否能用现有信息帮您上报问题.

**Comment by ToSsGirL06** [comment]
版本	Windows 10 专业版
版本号	22H2
安装日期	‎2022/‎8/‎16
操作系统内部版本	19045.4239
体验	Windows Feature Experience Pack 1000.19055.1000.0


蓝灯版本看不到但我是4月3日于官网下载的版本







Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月9日(星期二) 凌晨1:00
***@***.***&gt;;
抄送:&nbsp;"Jason ***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)





 
@ToSsGirL06 嗯嗯好的. 那请您告诉我您用的是哪一个系统? Window xx? Mac OS? Android? IOS xx?
 系统版本. 然后哪一个Lantern版本? 我也只能用这些信息帮您提交问题来修复.
 
但一般只需要去按帐号(account), 然后支持 (support) 就能提交报告/LOG了.
 
如果这里也闪退那就没事. 我看是否能用现有信息帮您上报问题.

**Comment by ToSsGirL06** [comment]
想问下这问题怎么解决，目前还是无法使用&nbsp;


Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年4月9日(星期二) 凌晨1:00
收件人: ***@***.***&gt;; 
抄送: "Jason ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)





 
@ToSsGirL06 嗯嗯好的. 那请您告诉我您用的是哪一个系统? Window xx? Mac OS? Android? IOS xx?
 系统版本. 然后哪一个Lantern版本? 我也只能用这些信息帮您提交问题来修复.
 
但一般只需要去按帐号(account), 然后支持 (support) 就能提交报告/LOG了.
 
如果这里也闪退那就没事. 我看是否能用现有信息帮您上报问题.
 
&mdash;

**Comment by MaxMKMax** [comment]
@ToSsGirL06 您好，非常抱歉影响了您的体验感。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@ToSsGirL06 请问您是无法登陆专业版还是闪退之类的问题? 如果无法登陆, 请您提供我账号邮箱和单据.

**Comment by ToSsGirL06** [comment]
我现在无法登陆AP，在开始-打开APP-网也不会弹出蓝灯已连接界面，右下角图标长灰，鼠标一放到右下图标上图标就消失。安装的最新版本蓝灯，系统情况之前发过你了。&nbsp;


Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年4月14日(星期天) 凌晨0:07
收件人: ***@***.***&gt;; 
抄送: "Jason ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)





 
@ToSsGirL06 请问您是无法登陆专业版还是闪退之类的问题? 如果无法登陆, 请您提供我账号邮箱和单据.
 
&mdash;

**Comment by ToSsGirL06** [comment]
***@***.***
单据指的是什么？
&nbsp;


Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年4月14日(星期天) 凌晨0:07
收件人: ***@***.***&gt;; 
抄送: "Jason ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)





 
@ToSsGirL06 请问您是无法登陆专业版还是闪退之类的问题? 如果无法登陆, 请您提供我账号邮箱和单据.
 
&mdash;

**Comment by MaxMKMax** [comment]
@ToSsGirL06 您好. 购买蓝灯的支付单据.

**Comment by ToSsGirL06** [comment]
我捣鼓好，删了个文件就ok了。
&nbsp;


Jason&nbsp;Wu
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年4月14日(星期天) 晚上10:26
收件人: ***@***.***&gt;; 
抄送: "Jason ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 打开蓝灯后电脑右下蓝灯图标消失 (Discussion #407)





 
@ToSsGirL06 您好. 购买蓝灯的支付单据.
 
&mdash;

**Reply by MaxMKMax** [reply]
@ToSsGirL06 很棒耶! 那就好. 谢谢您支持蓝灯!

**Comment by ToSsGirL06** [comment]
感谢你的支撑，但是如你所说的提供付款凭证，我买的都是2年的那种，找凭证挺麻烦的。希望改进售后方面的机制，其实我这个问题要不是自己歪打正着找到了解决方式，感觉得重装系统试试。因为你说的方法在我电脑上无法完成，就是像你说的发送log之类的
________________________________

**Comment by MaxMKMax** [comment]
@ToSsGirL06 嗯嗯. 同意您的建议. 我们会听取您的方案进行修改. 谢谢您的反馈和建议! 
那个log的话是我们拿来审计技术问题用的. 不然技术团队很难针对性的修复您遇到的特定问题. 我一般会拿票号和他们讨论问题, 然后他们再去修复.
